server/api/hooks.js

```diff
@@ -58,15 +58,18 @@ router.post('hooks.interactive', async ctx => {
   ctx.assertPresent(token, 'token is required');
   ctx.assertPresent(callback_id, 'callback_id is required');
 
-  if (token !== process.env.SLACK_VERIFICATION_TOKEN)
+  if (token !== process.env.SLACK_VERIFICATION_TOKEN) {
     throw new AuthenticationError('Invalid verification token');
+  }
 
-  const user = await User.findOne({
-    where: { service: 'slack', serviceId: data.user.id },
+  const team = await Team.findOne({
```

Comment:
We always trusted the data from Slack here, previously it was the user and now the team.